wiki/Manual.md

```diff
@@ -856,13 +853,11 @@ Supported rounding modes are:
 - `ROUND` – will round the last decimal digits to precision
 - `TRUNCATE`– will cut off the digits after certain precision
 
-The decimal precision counting mode is available in the `exchange.precisionMode` property.
+The decimal precision counting mode is available in the `exchange.precisionMode` property. <a id="exchangePrecisionMode"></a> Supported precision modes are:
```

Comment:
We should not use HTML anchors cause it will most likely break the RTD, which is auto-generated from this Manual, see:

- https://docs.ccxt.com
- https://ccxt.readthedocs.io/

I suggest replacing the anchor with Markdown (https://guides.github.com/features/mastering-markdown/) headers of proper depth, in this case:

```
##### Precision Mode
```
```